mlir/utils/performance/perfRunner.py

```diff
@@ -414,28 +429,39 @@ def fromCommandLine(cls, argv, arch):
             else:
                 continue
 
-        return cls(dataType, direction, layout, n, c, hi, wi, k, y, x,
+        if layout is None:
```

Comment:
More substantial comment: let's remove the `layout` variable all together, since we're now tracking filter, input, and output layouts seperately.